FrontEnd/src/Notifications/NotificationsPage.jsx

```diff
@@ -6,25 +6,22 @@ import "./NotificationPage.css";
 function NotificationsPage({ userData, vendorID }) {
   const userID = userData?.userID;
   const [userNotificationData, setUserNotificationData] = useState([]);
-  const [vendorNotificationData, setVendorNotificationData] = useState([]);
 
   useEffect(() => {
-    if (userID === undefined) return;
-    const unsubscribe = fetchNotifications(userID, (userAppointmentData) => {
-      setUserNotificationData(userAppointmentData);
+    if (userID === undefined){
+      return;
+    }
+    const unsubscribe = fetchNotifications(userID, (notificationData) => {
+      const userNotifications = notificationData.filter(
```

Comment:
I'm not sure I fully understands why we need to filter by notification.userID === userID here, do you mind explain?